config/locales/user_mailer/es.yml

```
@@ -126,6 +126,16 @@ es:
         persona, deberá llevar un comprobante de domicilio vigente si su
         dirección actual es distinta de la que aparece en su documento de
         identidad.
+    in_person_failed_suspected_fraud:
+      body:
+        help_center_html: Si necesita ayuda adicional, puede <a
+          href="%{help_center_url}">visitar nuestro Centro de Ayuda</a> o
+          ponerse en contacto con la agencia a la que intenta acceder.
+        intro: Entendemos que estaba intentando verificar su identidad a través de
+          Login.gov, sin embargo, su identidad no pudo ser verificada en la
```

```suggestion
          %{app_name}, sin embargo, su identidad no pudo ser verificada en la
```
We should always use `app_name` instead of hardcoding `Login.gov`